cg/apps/housekeeper/hk.py

```diff
@@ -482,3 +482,18 @@ def set_archive_retrieval_task_id(self, file_id: int, retrieval_task_id: int) ->
             raise ValueError(f"No Archive entry found for file with id {file_id}.")
         self._store.update_retrieval_task_id(archive=archive, retrieval_task_id=retrieval_task_id)
         self.commit()
+
+    def get_sample_sheets_from_latest_version(self, flow_cell_id: str) -> List[File]:
```

Comment:
Are we ever expecting more than one sample sheet from a version? Else it might be nice to have this method returning an Optional[File] instead.

Reply:
It may return the sample sheet and the archived sample sheet if the flow cell has both, which will be deprecated